networkx/utils/backends.py

```diff
@@ -339,22 +345,23 @@ def __new__(
         mutates_input=False,
         returns_graph=False,
     ):
-        """A decorator that makes certain input graph types dispatch to ``func``'s
-        backend implementation.
-
-        Usage can be any of the following decorator forms:
-        - @_dispatchable
-        - @_dispatchable()
-        - @_dispatchable(name="override_name")
-        - @_dispatchable(graphs="graph_var_name")
-        - @_dispatchable(edge_attrs="weight")
-        - @_dispatchable(graphs={"G": 0, "H": 1}, edge_attrs={"weight": "default"})
-        with 0 and 1 giving the position in the signature function for graph objects.
-        When edge_attrs is a dict, keys are keyword names and values are defaults.
-
-        The class attributes are used to allow backends to run networkx tests.
-        For example: `PYTHONPATH=. pytest --backend graphblas --fallback-to-nx`
-        Future work: add configuration to control these.
+        """A decorator function that is used to redirect the execution of ``func``
+        function to its backend implementation.
+
+        This decorator function dispatches to
+        a different backend implementation based on the input graph types, and it also
+        manages all the ``backend_kwargs``. Usage can be any of the following decorator
```

Comment:
Is this the first place we mention `backend_kwargs`? How should we talk about backend functions allowing additional keyword arguments? For that matter, do we talk about what `get_info` is able to return including `"additional_docs"`?

Also, should we mention `backend="my_backend"` to choose a bakcend when calling a dispatchable function?

Reply:
It is previously mentioned on the "Backends and Config" docs page but this is the first time mentioned on the `dispatchable` decorator docs page. On the "Backends and Config" docs page, I've made its explanation more detailed. Please look at it and LMK if something doesn't seem right there because I just tried to explain it by looking at the code in `__call__` of the `dispatchable` class, so I might be missing some context there. pls LMK if that looks fine.

I've also added the details of the get_info function as well.

Reply:
backend_kwargs: [here](https://github.com/networkx/networkx/blob/6abd4682ea2bddfb07427d8ef413cd8166ee29e5/networkx/utils/backends.py#L40)

get_info: [here](https://github.com/networkx/networkx/blob/6abd4682ea2bddfb07427d8ef413cd8166ee29e5/networkx/utils/backends.py#L161)